src/lambda.ts

```diff
@@ -29,10 +29,13 @@ export async function handler(): Promise<SendEmailResponse | string> {
 
 let shouldRun = (currentHour: number): boolean => config.RunHours.indexOf(currentHour) >= 0;
 
+//This lambda runs either after midnight or after 1am. Default to 1am in case we want to manually run later
+let currentHourString = () => new Date().getHours() === 0 ? "0000" : "0100";
```

Comment:
After reading the comment, but definitely not important
```js
() => `0${Date.now().getHours()}00`
```

Reply:
I want it to fall back on 0100 if we run it later on